x/tss/keeper/grpc_query.go

```diff
@@ -58,3 +62,71 @@ func (q Querier) AssignableKey(c context.Context, req *types.AssignableKeyReques
 
 	return &types.AssignableKeyResponse{Assignable: !assigned}, nil
 }
+
+// ValidatorKey returns a map of active multisig role key ids to pub keys of a validator
+func (q Querier) ValidatorKey(c context.Context, req *types.ValidatorKeyRequest) (*types.ValidatorKeyResponse, error) {
+	ctx := sdk.UnwrapSDKContext(c)
+
+	valAddress, err := sdk.ValAddressFromBech32(req.Address)
+	if err != nil {
+		return nil, err
+	}
+
+	validator := q.staker.Validator(ctx, valAddress)
+	if validator == nil {
+		return nil, fmt.Errorf("not a validator")
+	}
+
+	chains := q.nexus.GetChains(ctx)
+	keys := make(map[string]*types.ValidatorKeyResponse_Keys, 10)
+
+	keyRoles := []exported.KeyRole{exported.MasterKey, exported.SecondaryKey}
+
+	for _, chain := range chains {
+		for _, keyRole := range keyRoles {
+			if currentKey, found := q.keeper.GetCurrentKey(ctx, chain, keyRole); found {
+				if valKeys, ok := q.getSerializedMultisigKeys(ctx, currentKey, valAddress); ok {
+					keys[string(currentKey.ID)] = valKeys
+				}
+			}
+
+			if nextKey, found := q.keeper.GetNextKey(ctx, chain, keyRole); found {
+				if valKeys, ok := q.getSerializedMultisigKeys(ctx, nextKey, valAddress); ok {
+					keys[string(nextKey.ID)] = valKeys
+				}
+			}
+
+			oldActiveKeys, err := q.keeper.GetOldActiveKeys(ctx, chain, keyRole)
+			if err != nil {
+				return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, err.Error())
+			}
+
+			for _, oldActiveKey := range oldActiveKeys {
+				if valKeys, ok := q.getSerializedMultisigKeys(ctx, oldActiveKey, valAddress); ok {
+					keys[string(oldActiveKey.ID)] = valKeys
+				}
+			}
+		}
+	}
+
+	return &types.ValidatorKeyResponse{Keys: keys}, nil
+}
+
+func (q Querier) getSerializedMultisigKeys(ctx sdk.Context, key exported.Key, valAddress sdk.ValAddress) (*types.ValidatorKeyResponse_Keys, bool) {
+	if key.Type != exported.Multisig {
+		return nil, false
+	}
+
+	if pubkeys, ok := q.keeper.GetMultisigPubKeysByValidator(ctx, key.ID, valAddress); ok {
+		valKeys := make([][]byte, len(pubkeys))
+
+		for i, pubkey := range pubkeys {
+			wrappedKey := btcec.PublicKey(pubkey)
+			valKeys[i] = (&wrappedKey).SerializeCompressed()
```

Comment:
```suggestion
			valKeys[i] = wrappedKey.SerializeCompressed()
```

Reply:
Hmm, I see why it was complaining before